tests/pretty_format_java_test.py

```diff
@@ -48,6 +48,22 @@ def test__download_google_java_formatter_jar(ensure_download_possible, version):
     (
         (["invalid.java"], 1),
         (["pretty-formatted.java"], 0),
+        pytest.param(
+            [
+                "pretty-formatted.java",
+                "--google-java-formatter-jar-checksum=1e69f8b63c39a5124a8efb7bad213eb9ac03944339eb9580ae210b0c60565d9b",
```

Comment:
This test depends on the fact that we are assuming a very specific version of the tool (the default one in [here](https://github.com/macisamuele/language-formatters-pre-commit-hooks/blob/master/language_formatters_pre_commit_hooks/google_java_formatter.version))

This effectively will imply that updates of the default version will lead to test failures.

Can you please retain the test but explicitly pinning the version of reference?

Reply:
Sure